documentation/book/con-logging.adoc

```diff
@@ -0,0 +1,27 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-mirror-maker.adoc
+
+// Save the context of the assembly that is including this one.
+// This is necessary for including assemblies in assemblies.
+// See also the complementary step on the last line of this file.
+
+[id='assembly-logging-{context}']
+= Logging
+
+Use the `logging.internal` and `logging.external` properties to configure loggers and logger levels.
+
+You can set the log levels by specifying the loggers and their levels directly (inline) or use a custom (external) ConfigMap.
+
+Here we see examples of `inline` and `external` logging:
+
+include::snip-configmap-logging-{kind}.adoc[]
```

Comment:
Not necesarily for this PR. But we should add more detals about how to use the external logging. Such as how to create the config map, what should be inside etc. This is currently completly missing.

Reply:
Let's add to backlog